--- ../config.h	2017-09-30 00:28:10.804976974 +0200
+++ config.h	2017-10-03 01:46:45.992515699 +0200
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Fixed:pixelsize=18:antialias=false:autohint=false";
+static char font[] = "DejaVu Sans Mono:pixelsize=12:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -128,7 +128,7 @@
  * 6: Bar ("|")
  * 7: Snowman ("☃")
  */
-static unsigned int cursorshape = 2;
+static unsigned int cursorshape = 7;
 
 /*
  * Default colour and shape of the mouse cursor
